octoprint_displayprogress/static/js/displayprogress.js

```diff
@@ -0,0 +1,8 @@
+document.getElementById("symbol").addEventListener("change", updatePreview);
```

Comment:
Hm... IF this is necessary (I still think a preview should be achievable without custom JS, just depending on the existing settings view model), it should be a proper view model, not jquery using JS injected ninja style. Might interfere with existing data bindings, plus the `updatePreview` function is declared in the global namespace.